src/nvim/buffer.c

```diff
@@ -3565,26 +3559,29 @@ void get_rel_pos(win_T *wp, char_u *buf, int buflen)
         : (int)(above * 100L / (above + below)));
 }
 
-/*
- * Append (file 2 of 8) to "buf[buflen]", if editing more than one file.
- * Return TRUE if it was appended.
- */
-static int 
-append_arg_number (
-    win_T *wp,
-    char_u *buf,
-    int buflen,
-    int add_file                   /* Add "file" before the arg number */
+/// Append (file 2 of 8) to "buf[buflen]", if editing more than one file.
+/// Return true if it was appended.
+static bool append_arg_number(
+  win_T *wp,
+  char_u *buf,
+  int buflen,
+  int add_file                   /* Add "file" before the arg number */
 )
```

Comment:
e.g., I don't think lines such as 3565-3568 should be changed, especially since the arguments are still "old style" (not using `@param`, and comments aren't C++ style).